Assets/MRTK/Providers/WindowsMixedReality/XRSDK/Controllers/WindowsMixedRealityXRSDKArticulatedHand.cs

```diff
@@ -129,7 +129,10 @@ private void UpdateHandData(InputDevice inputDevice)
                                 Vector3 position = Vector3.zero;
                                 Quaternion rotation = Quaternion.identity;
 
-                                if (bone.TryGetPosition(out position) || bone.TryGetRotation(out rotation))
+                                bool positionAvailable = bone.TryGetPosition(out position);
+                                bool rotationAvailable = bone.TryGetRotation(out rotation);
+
+                                if (positionAvailable || rotationAvailable)
```

Comment:
If rotation or position isn't available, this will still update the position/rotation right? (i.e. I'm guessing it should just zero it out for that frame?)

Reply:
Yeah, the `TryGet*` code zeros it out before returning `false`.

Reply:
I guess we could cache the last known good value and reuse that? But for WMR at least, it's most likely we get all or nothing.

Reply:
Yeah I think either way it's kinda wonky - this is fine to go with (could still be worth having a comment because it looks suspect using a value when it failed. Calling out the intent here as being the least evil thing is useful).

Reply:
```suggestion
                                // If either position or rotation is available, use both pieces of data given.
                                // This might result in using a zeroed out position or rotation. Most likely,
                                // either both are available or both are unavailable.
                                if (positionAvailable || rotationAvailable)
```